docs/index.qmd

```diff
@@ -0,0 +1,7 @@
+---
+title: "Posit SDK for Python"
+---
+
+> A Pythonic interface for developers to work with Posit's professional products. It's lightweight and expressive.
```

Comment:
```suggestion
> A Pythonic interface for developers to work with Posit professional products. It's lightweight and expressive.
```

re: lightweight/expressive, see comment below re: statements that veer into marketing territory. These do seem like more industry standard descriptors so use your own judgement on this, but wanted to flag just in case :)

Reply:
Got it. This is just my best attempt, so I am happy to modify any of the language to what you think is best.